apps/007_ofsted_market_analysis/app.py

```diff
@@ -2,113 +2,498 @@
 import pandas as pd
 import plotly.express as px
 import matplotlib
+from pyodide.http import open_url
+
+def needs_coder(row, field):
+    if row[field] == 'Y':
+        return 'Yes'
+    elif (int(row[field])) > 0:
+        return 'Yes'
+    else:
+        return 'No'
+
+def plot_chart(data_frame, var_x, var_y, var_color, var_title, var_barmode, var_cdm=None, var_cat_orders=None):
+    fig = px.bar(data_frame,
+        x = var_x,
+        y = var_y,
+        color = var_color,
+        title = var_title,
+        barmode=var_barmode,
+        color_discrete_map=var_cdm,
+        category_orders=var_cat_orders)
+    st.plotly_chart(fig)
+
+
+data1 = open_url('https://raw.githubusercontent.com/Paggers92/PATCh_test/main/ofsted_benchmarking_app/Local%20authorities%20and%20regions.csv')
+regions = pd.read_csv(data1)
 
 st.markdown("[![Foo](https://github.com/data-to-insight/patch/blob/main/docs/img/contribute.png?raw=true)](https://www.datatoinsight.org/patch) \
              [![Foo](https://github.com/data-to-insight/patch/blob/main/docs/img/viewthecodeimage.png?raw=true)](https://github.com/data-to-insight/patch/blob/main/apps/007_ofsted_market_analysis/app.py)")
 
 # Title and description
-st.title('Ofsted Market Analysis - small providers by region')
-st.write('This tool analyses a list of social care providers as provided by Ofsted.')
-st.write('The user can filter on one or multiple regions, as well as a maximum threshold of settings per owner, to view a list of socal care settings that fit these criteria.')
+st.title('Ofsted Market Analysis')
+st.write('This tool analyses a list of social care settings providers as provided by Ofsted.')

```

Comment:
Can you just say a tiny bit more about what the analysis does, just a little break down like 'shows how many beds, who can care for children with specific needs etc', just a sentence or two